build.gradle

```diff
@@ -1,72 +1,47 @@
-buildscript {
+plugins {
+    id("java-library")
+    id("checkstyle")
+    id("eclipse")
+    id("jacoco")
+    id("maven-publish")
+    id("ru.vyarus.animalsniffer") version "1.5.3"
+    id("me.champeau.gradle.jmh") version "0.5.3"
+    id("com.github.hierynomus.license") version "0.15.0"
+    id("com.jfrog.artifactory") version "4.21.0"
+    id("biz.aQute.bnd.builder") version "5.3.0"
+    id("com.vanniktech.maven.publish") version "0.14.2"
+}
 
-  // Dependency versions
-  // ---------------------------------------
-  ext {
+ext {
     reactiveStreamsVersion = "1.0.3"
     junitVersion = "4.13.2"
     testNgVersion = "7.3.0"
     mockitoVersion = "3.8.0"
     jmhLibVersion = "1.21"
-    jmhGradleVersion = "0.5.3"
-    guavaVersion = "30.1.1-jre"
+    guavaVersion = "30.1-jre"
```

Comment:
Why going back to 30.1-jre here?

Reply:
I was working on an older branch. I will correct it